### DISCUSSION TITLE
Comment or annotate an article in freshRSS

### DISCUSSION THREAD

**Discussion by Strikelab** [FreshRSS/Extensions#160] [Ideas]
Hello,

Is there a way or an extension to annotate or comment an article in freshRSS ? 

Thanks for this really usefull application.

**Comment by math-GH** [comment]
As far as I see there is no extension for this use case.

I would suggest to have a look into bookmarking services. Here is an overview of services that could be used to have a quick sharing option from FreshRSS to the particular bookmark service

**Reply by Alkarex** [reply]
Indeed, not possible at the moment (but an extension could do it). I do consider adding proposing a basic functionality for that in the core though, but there are a few other features with higher priority on my personal TODO list before that :-)

**Comment by daniporr** [comment]
Have a look at https://github.com/wallabag/wallabag

**Comment by nycki93** [comment]
I support this idea! FreshRSS is the best reader I've seen so far that allows you to re-share your liked posts, and it would be wonderful if I could do the "share with comments" thing too! Like "social media" but, y'know, not!
